src/lib/input/input-container.spec.ts

```diff
@@ -550,6 +552,124 @@ describe('MdInputContainer', function () {
     expect(labelEl.classList).not.toContain('mat-float');
   });
 
+  describe('error messages', () => {
+    let fixture: ComponentFixture<MdInputContainerWithErrorMessages>;
+    let testComponent: MdInputContainerWithErrorMessages;
+    let containerEl: HTMLElement;
+
+    beforeEach(() => {
+      fixture = TestBed.createComponent(MdInputContainerWithErrorMessages);
+      fixture.detectChanges();
+      testComponent = fixture.componentInstance;
+      containerEl = fixture.debugElement.query(By.css('md-input-container')).nativeElement;
+    });
+
+    it('should not show any errors if the user has not interacted', () => {
+      expect(testComponent.formControl.pristine).toBe(true, 'Expected untouched form control');
```

Comment:
Do you mean pristine or untouched here? Those are separate FormControl states. It seems like you want `untouched`.